l1-contracts/contracts/common/Config.sol

```diff
@@ -106,5 +106,7 @@ address constant ETH_TOKEN_ADDRESS = address(1);
 
 bytes32 constant TWO_BRIDGES_MAGIC_VALUE = bytes32(uint256(keccak256("TWO_BRIDGES_MAGIC_VALUE")) - 1);
 
+address constant NATIVE_TOKEN_VAULT_VIRTUAL_ADDRESS = address(2);
```

Comment:
please add to the reader why we need to use native token vault virtual address, while usually for ADT we would use their true addresses